docs/syntax_and_semantics/annotations/built_in_annotations.md

```diff
@@ -1,16 +1,5 @@
 The Crystal standard library includes some pre-defined annotations:
 
-* [Link](#link)
-* [Extern](#extern)
-* [ThreadLocal](#threadlocal)
-* [Packed](#packed)
-* [AlwaysInline](#alwaysinline)
-* [NoInline](#noinline)
-* [ReturnsTwice](#returnstwice)
-* [Raises](#raises)
-* [CallConvention](#callconvention)
-* [Flags](#flags)
-
```

Comment:
Note: this manual list is obsoleted by the automatic right sidebar

Preview: https://oprypin.github.io/crystal-book/syntax_and_semantics/annotations/built_in_annotations.html